src/runtime/graph/graph_runtime.cc

```diff
@@ -508,8 +554,9 @@ void GraphRuntime::SetupOpExecs() {
       uint32_t eid = this->entry_id(nid, index);
       args.push_back(*(data_entry_[eid].operator->()));
     }
-    CHECK_EQ(inode.op_type, "tvm_op")
-        << "Can only take tvm_op as op";
+    CHECK(inode.op_type == "tvm_op" || inode.op_type == "device_copy_op")
```

Comment:
since we are already using __copy for cross-device copy, we just need to make sure op_type is "tvm_op"